lib/views/bottom-status.js

```diff
@@ -9,24 +9,50 @@ class BottomStatus extends HTMLElement{
     this.iconSpan.classList.add('icon')
     this.appendChild(this.iconSpan)
 
-    this.count = 0
+    this._linting = false
+    this._count = 0
```

Comment:
Underscore is only a place holder here, we have two options here

1. Remove getter/setters and underscores and use property names as keys and have setters and getters like `setSomething` and `getSomething`
2. Or we keep underscore for those who really need it, and use es6 setters/getters